.github/workflows/publish.yml

```diff
@@ -28,9 +28,53 @@ jobs:
           yarn test
           yarn package
 
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      # Build and publish Docker runner (for GitLab CI/CD, etc.)
+      - name: Publish Docker image
+        working-directory: ./docker
+        run: |
+          full_version="1.2.3"
```

Comment:
Ouch, that's a bad copy-paste from testing the script! Well spotted! 👍

It's supposed to be the tag for the release, I'll fix it. The intention for now is to publish both with the same version number, since the majority of the code is the same, and since users typically won't be impacted even if there are some patch/minor releases which aren't strictly necessary for one of the artifacts (as they can just specify the major version).